[Codechange] Some screenshot taking improvements

ulteq opened the pull request:
* Reduces the screenshot key bounce time by half
* Adds time stamps to the screenshot file name
* Automatically hides screen notifications before another screenshot is taken
* Gets rid of the camera stuttering when taking a screenshot (introduced by: 1ff982a4b8bee32d6caf715503c8c31b52350c63)
* Prevents recording partially updated trucks when taking a screenshot (introduced by: 1ff982a4b8bee32d6caf715503c8c31b52350c63)

Fixes: #360
Fixes: #473

only-a-ptr commented:
On windows, it behaves funny... all code is executed without exception or any error indication, the file path is valid: ```"C:\\Users\\only_a_ptr\\DOCUME~1\\Rigs of Rods 0.4\\screenshot_2016-01-30_22:37:58_1.jpg"``` yet the screenshot is not saved. Problem: colons are illegal in filenames on windows. OGRE obviously doesn't do error handling right here.

ulteq commented:
> colons are illegal in filenames on windows

Fixed: e902eb9

> However, taking screenshot causes nasty stuttering of camera. It's very annoying while driving.

Fixed: b88a1fc

only-a-ptr commented:
:+1:
